examples/01-filter/image-representations.py

```diff
@@ -0,0 +1,223 @@
+#
+# For context, we first show the input data array.
+
+points_volume.point_data['Data']
+
+################################################################################
+# Now apply the filter and print the result.
+
+points_ithresh = points_volume.image_threshold(2)
+points_ithresh.point_data['Data']
+
+################################################################################
+# The filter returns binary point data as expected. Values equal to or greater
+# or than the threshold of ``2`` are ones, and less than the threshold are zeros.
+#
+# However, when we plot it the point values are linearly interpolated. For
+# visualizing binary data, this interpolation is not desirable.
+
+points_ithresh.plot(show_edges=True)
+
+################################################################################
+# To better visualize the result, convert the points image returned by the
+# filter to a cell representation with :meth:`~pyvista.ImageDataFilters.points_to_cells`
+# before plotting.
+
+points_ithresh_as_cells = points_ithresh.points_to_cells()
+points_ithresh_as_cells.plot(show_edges=True)
+
+################################################################################
+# The binary data is now correctly visualized as binary data.
+
+################################################################################
+# Cell Filters with Image Data
+# ----------------------------
+# Use a cell representation of the image when working with cell-based filters
+# such as :meth:`~pyvista.DataSetFilters.threshold`. If the image only has point
+# data, use :meth:`~pyvista.ImageDataFilters.points_to_cells` to re-mesh the
+# input first. Here, we reuse the cell-based image created earlier.
+#
+# For context, we first show the input data array.
+
+cells_volume.cell_data['Data']
+
+################################################################################
+# Now apply the filter and print the result.
+
+cells_thresh = cells_volume.threshold(2)
+cells_thresh.cell_data['Data']
+
+################################################################################
+# When the input is cell data, this filter returns seven discrete values greater
+# than or equal to the threshold value of ``2`` as expected.
+#
+# If we plot the result, the cells also produce the correct visualization.
+
+cells_thresh.plot(show_edges=True)
+
+################################################################################
+# However, if we apply the same filter to a point-based representation of the
+# image, the filter does not produce the desired result.
+
+points_thresh = points_volume.threshold(2)
+points_thresh.point_data['Data']
+
+################################################################################
+# In this case, since the points image only has a single cell, the filter has no
+# effect on the data array's values. The thresholded values are the same as the
+# input values.
```

Comment:
```suggestion
# In this case, since the image of the point only has a single cell, the filter has no
# effect on the data array's values. The thresholded values are the same as the
# input values.
```